collector/aspnet_aspnetapplications.go

```diff
@@ -0,0 +1,1239 @@
+	MembershipAuthenticationSuccess       *prometheus.Desc
+	OutputCacheEntries                    *prometheus.Desc
+	OutputCacheHitRatio                   *prometheus.Desc
+	OutputCacheHits                       *prometheus.Desc
+	OutputCacheMisses                     *prometheus.Desc
+	OutputCacheTrims                      *prometheus.Desc
+	OutputCacheTurnoverRate               *prometheus.Desc
+	PercentManagedProcessorTimeestimated  *prometheus.Desc
+	PipelineInstanceCount                 *prometheus.Desc
+	RequestBytesInTotal                   *prometheus.Desc
+	RequestBytesInTotalWebSockets         *prometheus.Desc
+	RequestBytesOutTotal                  *prometheus.Desc
+	RequestBytesOutTotalWebSockets        *prometheus.Desc
+	RequestErrorEventsRaised              *prometheus.Desc
+	RequestErrorEventsRaisedPerSec        *prometheus.Desc
+	RequestEventsRaised                   *prometheus.Desc
+	RequestEventsRaisedPerSec             *prometheus.Desc
+	RequestExecutionTime                  *prometheus.Desc
+	RequestsDisconnected                  *prometheus.Desc
+	RequestsExecuting                     *prometheus.Desc
+	RequestsExecutingWebSockets           *prometheus.Desc
+	RequestsFailed                        *prometheus.Desc
+	RequestsFailedWebSockets              *prometheus.Desc
+	RequestsInApplicationQueue            *prometheus.Desc
+	RequestsNotAuthorized                 *prometheus.Desc
+	RequestsNotFound                      *prometheus.Desc
+	RequestsPerSec                        *prometheus.Desc
+	RequestsRejected                      *prometheus.Desc
+	RequestsSucceeded                     *prometheus.Desc
+	RequestsSucceededWebSockets           *prometheus.Desc
+	RequestsTimedOut                      *prometheus.Desc
+	RequestsTotal                         *prometheus.Desc
+	RequestsTotalWebSockets               *prometheus.Desc
+	RequestWaitTime                       *prometheus.Desc
+	SessionsAbandoned                     *prometheus.Desc
+	SessionsActive                        *prometheus.Desc
+	SessionSQLServerconnectionstotal      *prometheus.Desc
+	SessionStateServerconnectionstotal    *prometheus.Desc
+	SessionsTimedOut                      *prometheus.Desc
+	SessionsTotal                         *prometheus.Desc
+	TransactionsAborted                   *prometheus.Desc
+	TransactionsCommitted                 *prometheus.Desc
+	TransactionsPending                   *prometheus.Desc
+	TransactionsPerSec                    *prometheus.Desc
+	TransactionsTotal                     *prometheus.Desc
+	ViewstateMACValidationFailure         *prometheus.Desc
+}
+
+// Newaspnet_aspnetapplicationsCollector ...
+func Newaspnet_aspnetapplicationsCollector() (Collector, error) {
+	const subsystem = "aspnet_aspnetapplications"
+	return &aspnet_aspnetapplicationsCollector{
+		AnonymousRequests: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "anonymous_requests"),
+			"(AnonymousRequests)",
+			nil,
```

Comment:
As noted below, check if there shouldn't be a label for application on all of these.